index.html

```diff
@@ -48,32 +48,71 @@ <h2>Advancing the future of the internet</h2>
         <!-- Benefits section -->
         <section class="mzp-l-content benefits-section">
             <div class="benefits-text-wrapper">
-                <h2><span class="dark-highlight">Benefits & Perks</span></h2>
+                <h2><span class="dark-highlight">Why Join Us?</span></h2>
             </div>
-            <div class="mzp-t-picto-center">
+            <div class="why-join-container">
+                <h3 class="why-join-subheader">For Builders</h3>
                 <section class="mzp-c-picto">
                     <div class="mzp-c-picto-image">
-                        <img src="/img/handshake.svg" width="90" alt="">
+                        <img src="/img/handshake.svg" height="75" alt="">
                     </div>
                     <h3 class="mzp-c-picto-heading">No-strings attached grant – we don’t take equity!</h3>
                 </section>
-            </div>
-            <div class="mzp-t-picto-center">
                 <section class="mzp-c-picto">
                     <div class="mzp-c-picto-image">
-                        <img src="/img/lightbulb.svg" width="90" alt="">
+                        <img src="/img/lightbulb.svg" height="75" alt="">
                     </div>
                     <h3 class="mzp-c-picto-heading">Stipends for learning and networking (e.g. conferences)</h3>
                 </section>
-            </div>
-            <div class="mzp-t-picto-center">
                 <section class="mzp-c-picto">
                     <div class="mzp-c-picto-image">
-                        <img src="/img/world.svg" width="90" alt="">
+                        <img src="/img/world.svg" height="75" alt="">
                     </div>
                     <h3 class="mzp-c-picto-heading">Access to Mozilla’s ecosystem of people, partners, and organizations</h3>
                 </section>
-            </div>
+            </div> <!-- end of why us section -->
+            <div class="why-join-container">
+                <h3 class="why-join-subheader">For Collaborators</h3>
+                <section class="mzp-c-picto">
+                    <div class="mzp-c-picto-image">
+                        <img src="/img/grant.svg" height="75" alt="">
+                    </div>
+                    <h3 class="mzp-c-picto-heading">Collaborate with existing MIECO builders while receiving a 10-100k grant</h3>
+                </section>
+                <section class="mzp-c-picto">
+                    <div class="mzp-c-picto-image">
+                        <img src="/img/server.svg" height="75" alt="">
+                    </div>
+                    <h3 class="mzp-c-picto-heading">Learn from open source partners and orgs in the Mozilla ecosystem</h3>
```

Comment:
**Nit:** I suggest we change `org` to `organizations`. Better for screen readers and for folks who may not use English as a second language. Best to make it as simple as possible for everyone.